implementations/README.md

```diff
@@ -1,9 +1,30 @@
 # Examples of implementations for metrics
 
 Notebooks, built on top of data collected by Perceval,
-for illustrating how to compute metrics:
+for illustrating how to compute different metrics of the Evolution Working Group. The notebooks will be explanatory in regard to the analysis performed. They should help the user understand how to calculate these metrics, as well as to understand how the metric is defined. 
+  
+The contents of this directory are the notebooks, as well as some scripts in the scripts directory. The scripts directory is a package containing the following modules:  
 
-* [Code Commits](Code_Commits.ipynb),
+- **Metric.py**:     
+This file has the root class, Metric. All other classes from the . It takes JSON data collected by Perceval and converts it into a form easier to analyze: like a dataframe, or a list of dictionaries.  
```

Comment:
I think you have missed something by mistake. :sweat_smile:

Reply:
Good catch! Thanks a lot, @vchrombie.